### DISCUSSION TITLE
scammer!

### DISCUSSION THREAD

**Discussion by KIWIONER** [community/community#33997] [Other Feature Feedback, Questions, & Ideas]
Please, 

This user, has created a website for a scamming group https://ee9pool.com/
this group is stealing money from SAFEPAL, a DAPP.

1. https://github.com/emn178

* [js-sha3]{@link https://github.com/emn178/js-sha3}
 *
 * @version 0.8.0
 * @author Chen, Yi-Cyuan [emn178@gmail.com]
 * @copyright Chen, Yi-Cyuan 2015-2018
 * @license MIT
 
I found the information on the inspector from Google.
I got scammed from this group and I am trying to recuperate my money. 
Please if anyone can help me, denunciate this person and help me find  more information will be amazing.
I want to report this user to github but I am new in this platform.
Thank you so much for your collaboration and kindness.

**Comment by EleganceAdel** [comment]
This project is only an open source encode/decoder
